weaviate/collections/batch/base.py

```diff
@@ -328,19 +342,19 @@ def start(self) -> "_Batch":
 
         return self
 
-    def shutdown(self) -> None:
+    def shutdown(self: B) -> None:
         """Shutdown the BatchExecutor."""
-        if not (self.__executor is None or self.__executor._is_shutdown()):
-            self.__executor.shutdown()
+        # if not (self.__executor is None or self.__executor._is_shutdown()):
+        #     self.__executor.shutdown()
```

Comment:
Because the batch executor is handled outside of this class and should not be shutdown by the end of this flow. It can be safely removed, however.

The general idea is to have one batch executor that works for both `client.batch` and `collection.batch` so that a user doesn't accidentally spawn many (potentially hundreds) of ThreadPoolExecutors in each of their collections. This way, there's a single pool that handles all allowed threads minimising associated overheads.

Reply:
makes sense - can you remove it? (If you already did just resolve this)